std/node/module_test.ts

```diff
@@ -69,3 +69,15 @@ Deno.test("requireStack", function () {
     assertStringIncludes(e.stack, "/tests/cjs/cjs_throw.js");
   }
 });
+
+Deno.test("requireFileInSymlinkDir", async () => {
+  // I don't think symlinks would hold up too well through Git, so create it at runtime
```

Comment:
We have symlinks in the repo (`cli/tests/symlink_to_subdir`), I think it's preferable solution to creating at runtime